### DISCUSSION TITLE
get 请求，定义参数类型的问题

### DISCUSSION THREAD

**Discussion by 840326521** [pallets/flask#5509] [Q&A]
为什么在 get 请求中，设置某个参数类型为 int 类型。
![image](https://github.com/pallets/flask/assets/38832100/6f2ad22b-63b1-4ea1-8567-975d146c061c)

而在浏览器这样请求时，却报 404  错误。
![image](https://github.com/pallets/flask/assets/38832100/0c28c5c7-bb27-4669-b877-c90fb7b0e68a)

Flaks 版本
![image](https://github.com/pallets/flask/assets/38832100/244e1622-2ea0-42ef-98f9-7710c55d83f9)

**Comment by davidism** [comment]
The `int` and `float` converters only match positive values by default. https://werkzeug.palletsprojects.com/en/3.0.x/routing/#werkzeug.routing.IntegerConverter You can pass an argument to change that: `<int(signed=True):age>`
